R/rdeephaven/tests/testthat/test_table_handle_wrapper.R

```diff
@@ -0,0 +1,173 @@
+library(testthat)
+library(rdeephaven)
+
+setup <- function() {
+
+    df1 <- data.frame(string_col = c("I", "am", "a", "string", "column"),
+                int_col    = c(0, 1, 2, 3, 4),
+                dbl_col    = c(1.65, 3.1234, 100000.5, 543.234567, 0.00))
+
+    df2 <- data.frame(col1 = rep(3.14, 100),
+                        col2 = rep("hello!", 100),
+                        col3 = rnorm(100))
+
+    df3 <- data.frame(matrix(rnorm(10 * 1000), nrow = 10))
+
+    df4 <- data.frame(time_col = seq.POSIXt(as.POSIXct(Sys.Date()), as.POSIXct(Sys.Date()+30), by = "1 sec")[250000],
+                    bool_col = sample(c(TRUE, FALSE), 250000, TRUE), 
+                    int_col = sample(0:10000, 250000, TRUE))
+
+    # in order to test TableHandle, we need to have tables on the server that we know everything about.
+    # thus, we have to push these created tables to the server and get TableHandles to each of them.
+    # thus, we depend on the correctness of client$import_table(), ClientOptions$new(), and Client$new()
+    
+    # set up client
+    client_options <- ClientOptions$new()
```

Comment:
Will this work in CI with the new PSK default?

Reply:
Ticket: https://github.com/deephaven/deephaven-core/issues/4091